package.json

```diff
@@ -62,6 +62,7 @@
   "dependencies": {
     "invariant": "^2.2.1",
     "lodash": "^4.13.1",
```

Comment:
Do we still need `lodash` then? I don't have familiarity with `lodash-es` and the change in `.babelrc`, maybe you could give some context?